README.md

```diff
@@ -63,7 +63,8 @@ Also, `nerdctl` might be potentially useful for debugging Kubernetes clusters, b
 
 - `nerdctl ps`
   - `-a, --all`: Show all containers (default shows just running)
-  - `--no-trunc`
+  - `--no-trunc`: Don't truncate output
+  - `-q, --quiet`: Only display numeric IDs
```

Comment:
not numeric 😛 

`docker ps --help` describes `-q, --quiet`  as "Only display container IDs"

Reply:
Strange! I am on mac and this is what I see

```
SM-smahajan628:images smahajan$ docker ps --help

Usage:	docker ps [OPTIONS]

List containers

Options:
  -a, --all             Show all containers (default shows just running)
  -f, --filter filter   Filter output based on conditions provided
      --format string   Pretty-print containers using a Go template
  -n, --last int        Show n last created containers (includes all states) (default -1)
  -l, --latest          Show the latest created container (includes all states)
      --no-trunc        Don't truncate output
  -q, --quiet           Only display numeric IDs
  -s, --size            Display total file sizes
```

Reply:
Lmk if you still want me to change it to `Only display container IDs`

Reply:
Interesting , may be your CLI is old?

![image](https://user-images.githubusercontent.com/9248427/101610774-0920c480-3a4c-11eb-93a5-0ed9246a7eac.png)

Reply:
Seems changed in v20.10.0-beta1 https://github.com/docker/cli/commit/9d9568263fbaad7253354931f9b3d1b9446d0c12 .

"Only display numeric IDs" is the new and correct form.

Reply:
Fixed.